### DISCUSSION TITLE
求教高手DatePicker日期选择框样式乱了是怎么回事？

### DISCUSSION THREAD

**Discussion by xiaomaodml** [ant-design/ant-design#34038] [Q&A]
![image](https://user-images.githubusercontent.com/39544907/153815641-d2ad51c2-4587-45f7-b109-8fe45d36abc2.png)
没有指定额外的样式，弹框的样式就成这样了。其他控件如Input、Cascader、Button等样式都正常。求教高手。

**Comment by xrkffgg** [comment]
提供个复现？

**Reply by xiaomaodml** [reply]
咋提供呢？就只是在页面中直接引入，没有做其他的样式设置。也没有在样式中覆写组件的类名。

**Reply by xrkffgg** [reply]
https://codesandbox.io/s/antd-reproduction-template-6e93z

**Reply by xrkffgg** [reply]
什么浏览器?

**Reply by xiaomaodml** [reply]
用的chrome和edge。组件库也是4.18.6。刚刚我直接在项目index页面直接渲染该组件也是一样。会不会是升级组件库版本导致的。我待会试试新建一个空项目，看会不会有问题。

**Reply by xiaomaodml** [reply]
找到原因了，是在项目index.less文件中定义了标签的一些基础样式，display:block，结果导致污染了antD的样式。
